### PR TITLE
Header8

### PR DESCRIPTION
**Pull request by BurcuAcan**

### CODE REVIEW

File: editor-components/header/header8/header8.module.scss [hunk 1]
```diff
@@ -1,214 +1,273 @@
...
+  }
+  .max-content, .max-content-no-image {
     width: 100%;
     position: absolute;
-    bottom: 90px;
+    bottom: 0px;
     left: 50%;
     transform: translateX(-50%);
-    
+    padding: 20px 50px;
```
Comment (Ayazelar) [inline editor-components/header/header8/header8.module.scss:87]: cant you use gap value?

File: editor-components/header/header8/header8.module.scss [hunk 2]
```diff
@@ -1,214 +1,273 @@
...
+        z-index: 20;
+        transition: transform 0.3s ease, border-color 0.3s ease;
+      }
+      &::after {
+        content: '';
+        position: absolute;
+        top: 50%;
+        right: -40px;
+        width: 40px;
+        height: 1px;
```
Comment (Ayazelar) [inline editor-components/header/header8/header8.module.scss:191]: also here. can't you use gap value ?

File: editor-components/header/header8/header8.module.scss [hunk 3]
```diff
@@ -1,214 +1,273 @@
...
-    .letter{
-      transform: translateY(-2em);
+@container (max-width: #{$composer-tablet-width}){
+  .container{
+    .carousel, .carousel-no-image {
+      .arrow-next, .arrow-next-no-image {
+        right: 40px;
+      }
+      .arrow-prev, .arrow-prev-no-image{
+        left: 40px;
```
Comment (Ayazelar) [inline editor-components/header/header8/header8.module.scss:229]: gap value ?

File: editor-components/header/header8/header8.module.scss [hunk 4]
```diff
@@ -1,214 +1,273 @@
...
+      }
     }
-  }
-  &--to_left{
-    .letter{
-      transform: translateX(2em);
+    .max-content, .max-content-no-image{
+      padding: 20px 40px;
```
Comment (Ayazelar) [inline editor-components/header/header8/header8.module.scss:233]: gap value ?

File: editor-components/header/header8/header8.module.scss [hunk 5]
```diff
@@ -1,214 +1,273 @@
...
-  &--to_middle{
-    .letter{
-      &:nth-child(2n) {
-        transform: translateX(2000px);
+}
+@container (max-width: #{$composer-phone-width}){
+  .container{
+    .carousel, .carousel-no-image{
+      .arrow-next, .arrow-next-no-image {
+        right: 20px;
```
Comment (Ayazelar) [inline editor-components/header/header8/header8.module.scss:241]: gap value ?

File: editor-components/header/header8/header8.module.scss (hunk 1 above)
Comment (BurcuAcan) [reply editor-components/header/header8/header8.module.scss:87]: I use this variable for the static container value, so I think I should not use the gap variable. We also talked about this issue.

File: editor-components/header/header8/header8.module.scss (hunk 3 above)
Comment (BurcuAcan) [reply editor-components/header/header8/header8.module.scss:229]: these are same reason

File: editor-components/header/header8/header8.module.scss (hunk 2 above)
Comment (BurcuAcan) [reply editor-components/header/header8/header8.module.scss:191]: these are same reason

**Review by Ayazelar — APPROVED** [review]

**Pull request merged**